app/controllers/application_controller.rb

```diff
@@ -7,11 +7,16 @@ class ApplicationController < ActionController::Base
 
   CONST_CREATE = 'create'.freeze
   def configure_devise_permitted_parameters
-    registration_params = [:name, :email, :password, :password_confirmation, :role, :department_id, company_attributes: [:name, :description]]
+    registration_params = [:name, :email, :password, :password_confirmation, :role, :department_id, company_attributes: [:name, :description, :subdomain]]
     if params[:action] == CONST_CREATE
       devise_parameter_sanitizer.for(:sign_up) do |parameters|
         parameters.permit(registration_params)
       end
     end
   end
+
+  def current_company
+    Company.find_by_subdomain request.subdomain
```

Comment:
cant it be "current_employee.company" ?

Reply:
no I am getting subdomain from the url.

Reply:
if a user is logged in and tries to open some other company's page. will he be able to?

Reply:
nah he can't will be logged out